Extension/package.json

```diff
@@ -282,2546 +282,2593 @@
+            "type": [
+              "integer",
+              "null"
+            ],
+            "markdownDescription": "%c_cpp.configuration.maxConcurrentThreads.markdownDescription%",
+            "default": null,
+            "minimum": 1,
+            "maximum": 32,
+            "scope": "machine"
+          },
+          "C_Cpp.maxCachedProcesses": {
+            "type": [
+              "integer",
+              "null"
+            ],
+            "markdownDescription": "%c_cpp.configuration.maxCachedProcesses.markdownDescription%",
+            "default": null,
+            "minimum": 0,
+            "maximum": 256,
+            "scope": "machine"
+          },
+          "C_Cpp.maxMemory": {
+            "type": [
+              "integer",
+              "null"
+            ],
+            "markdownDescription": "%c_cpp.configuration.maxMemory.markdownDescription%",
+            "default": null,
+            "minimum": 256,
+            "maximum": 65536,
+            "scope": "machine"
+          },
+          "C_Cpp.clang_format_path": {
+            "type": "string",
+            "markdownDescription": "%c_cpp.configuration.clang_format_path.markdownDescription%",
+            "scope": "machine-overridable"
+          },
+          "C_Cpp.clang_format_style": {
+            "type": "string",
+            "default": "file",
+            "markdownDescription": "%c_cpp.configuration.clang_format_style.markdownDescription%",
+            "scope": "resource"
+          },
+          "C_Cpp.formatting": {
+            "type": "string",
+            "enum": [
+              "clangFormat",
+              "vcFormat",
+              "default",
+              "disabled"
+            ],
+            "markdownEnumDescriptions": [
+              "%c_cpp.configuration.formatting.clangFormat.markdownDescription%",
+              "%c_cpp.configuration.formatting.vcFormat.markdownDescription%",
+              "%c_cpp.configuration.formatting.Default.markdownDescription%",
+              "%c_cpp.configuration.formatting.Disabled.markdownDescription%"
+            ],
+            "default": "default",
+            "description": "%c_cpp.configuration.formatting.description%",
+            "scope": "resource"
+          },
+          "C_Cpp.clang_format_fallbackStyle": {
+            "type": "string",
+            "default": "Visual Studio",
+            "markdownDescription": "%c_cpp.configuration.clang_format_fallbackStyle.markdownDescription%",
+            "scope": "resource"
+          },
+          "C_Cpp.clang_format_sortIncludes": {
+            "type": [
+              "boolean",
+              "null"
+            ],
+            "enum": [
+              true,
+              false,
+              null
+            ],
+            "default": null,
+            "markdownDescription": "%c_cpp.configuration.clang_format_sortIncludes.markdownDescription%",
+            "scope": "resource"
+          },
+          "C_Cpp.intelliSenseEngine": {
```

Comment:
From here down, a lot of these settings go in the "IntelliSense" section.  Maybe not loggingLevel or preferredPathSeparator.

Reply:
also, C_Cpp.experimentalFeatures